ui/partials/main/jobs.html

```diff
@@ -25,10 +25,6 @@
     </span>
     <th-result-counts class="result-counts"></th-result-counts>
     <span class="result-set-buttons">
-      <a class="btn btn-sm btn-resultset test-view-btn"
```

Comment:
Could you comment this block out instead, and reference the bug number? :-)

Reply:
Done.